src/styles/breakpoints.scss

```diff
@@ -0,0 +1,74 @@
+// this mixin creates the main breakpoints
+// to create a breakpoint, format like this: 
+// @include bp(large) {
+//    styles go here
+//  }
+// custom breakpoints would be in this format:
+// @include bp('(max-width: 30px) and (min-width: 20px)')
+
+$breakpoints: (
+	xsmall: 390px,
+	small: 640px,
+	medium: 1024px,
+	large: 1440px
+);
```

Comment:
you might as well remove bp() once you move to break().  this is scoped solely to this package so no need.